apps/client/ui/Range/Range.tsx

```diff
@@ -0,0 +1,101 @@
+import { useState, useEffect, useRef } from 'react'
+import type { FC } from 'react'
+import 'rc-slider/assets/index.css';
+import Slider from 'rc-slider';
+
+type CustomRangeProps = {
+  min?: number
+  max?: number
+  step?: number
+  defaultValue?: number[]
+  className?: string
+};
+
+const sliderStyles = ''
+
+export const CustomRange: FC<CustomRangeProps> = ({
+  min = 0,
+  max = 10,
+  step = 1,
+  defaultValue = [0, 10],
+  className,
+  ...props
+}) => {
+  const [values, setValues] = useState<number[]>(defaultValue);
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  const handleOnChange = (newValues: number[]) => {
+    setValues(newValues);
+  };
+
+  const handleTooltipPosition = () => {
+    if (!containerRef.current) return;
+
+    const [leftHandle, rightHandle] = Array.from(containerRef.current.getElementsByClassName('rc-slider-handle')) as HTMLElement[];
+    const [leftTooltip, rightTooltip] = Array.from(containerRef.current.getElementsByClassName('slider-tooltip')) as HTMLElement[];
+
+    if (leftHandle && leftTooltip && rightHandle && rightTooltip) {
+      const containerRect = containerRef.current.getBoundingClientRect();
+      const leftHandleRect = leftHandle.getBoundingClientRect();
+      const rightHandleRect = rightHandle.getBoundingClientRect();
+      const rightTooltipRect = rightTooltip.getBoundingClientRect();
+
+      const leftOffset = leftHandleRect.left - containerRect.left - leftHandleRect.width;
+      leftTooltip.style.left = `${leftOffset}px`;
+
+      const rightOffset = rightHandleRect.left - containerRect.left + rightHandleRect.width - rightTooltipRect.width;
+      rightTooltip.style.left = `${rightOffset}px`;
+    }
+  };
+
+  useEffect(() => {
+    handleTooltipPosition();
+    window.addEventListener('resize', handleTooltipPosition);
+    return () => {
+      window.removeEventListener('resize', handleTooltipPosition);
+    };
+  }, [values]);
+
+  return (
+    <div className='min-w-[180px] mt-12 mx-6 mb-0 h-9' ref={containerRef} {...props}>
+      <Slider
+        range
+        min={min}
+        max={max}
+        step={step}
+        allowCross={false}
+        defaultValue={values}
+        onChange={handleOnChange}
+        trackStyle={[{ backgroundColor: 'var(--primary-color)', height: '4px' }]}
+        railStyle={{ backgroundColor: 'var(--dark-grey)', height: '4px' }}
+        handleStyle={[
+          {
+            backgroundColor: 'var(--primary-color)',
+            height: '14px',
+            width: '14px',
+            border: 'none',
+          },
+          {
+            backgroundColor: 'var(--primary-color)',
+            height: '14px',
+            width: '14px',
+            border: 'none',
+          },
+        ]}
+        className='w-full p-10'
+      />
+      <div className="flex mt-[5px] justify-between">
```

Comment:
It's good practice with Tailwind to trying to use it's declared classes if possible - so if there is some similar value in Tailwind it's better to use it (even if it will not be pixel perfect than - another option is to declare needed values in the theme and then use them with prepared classes). Using declared classes makes the code more consistent and easier to maintain - it's big advantage of working with Tailwind compared to other CSS solutions 👌